package.json

```diff
@@ -94,11 +97,19 @@
     "unzip": "^0.1.11"
   },
   "devDependencies": {
+    "@babel/cli": "^7.1.5",
+    "@babel/core": "^7.1.6",
+    "@babel/node": "^7.0.0",
+    "@babel/plugin-proposal-class-properties": "^7.1.0",
+    "@babel/plugin-transform-runtime": "^7.1.0",
+    "@babel/polyfill": "^7.0.0",
+    "@babel/preset-env": "^7.1.6",
+    "@babel/preset-react": "^7.0.0",
```

Comment:
Do you need to specify all these deps? I thought that just adding preset-react would have it depend on the right stuff, but I may be wrong.

Reply:
It seems to be the case, yeah